annotations-runtime/build.gradle

```diff
@@ -9,19 +9,19 @@ subprojects {
 
         compile project(':annotations')
         compile project(":webscripts")
+        compile project(':annotations-runtime-schedule') // schedule abstraction
 
         compileOnly "org.apache.felix:org.apache.felix.framework:${project.ext.felixVersion}"
 
         compileOnly "org.slf4j:slf4j-api"
         compileOnly("org.alfresco.surf:spring-webscripts") { 
-            exclude group: "org.quartz-scheduler"
+            exclude group: "org.schedule-schedule"
```

Comment:
Is this correct?

Reply:
Good catch, that's some IntelliJ refactoring trying to be helpful :-)